logstash-core-queue-jruby/build.gradle

```diff
@@ -0,0 +1,111 @@
+group 'org.logstash'
+version = '6.0.0-alpha1'
```

Comment:
version should be handled programmatically via `version.yml` as previously mentioned